pom.xml

```diff
@@ -265,6 +265,23 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>3.3.0</version>
+                <executions>
+                    <execution>
+                        <id>copy-dependencies</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>copy-dependencies</goal>
+                        </goals>
+                        <configuration>
+                            <outputDirectory>${project.build.directory}/lib</outputDirectory>
+                        </configuration>
```

Comment:
The `configuration` section is most definitely insufficient here. If you look at the resulting directory:

```
[rhusar@ribera jgroups-aws]$ ll target/lib/
total 30296
-rw-r--r--  1 rhusar  staff  1244287 Jun  4 17:26 assertj-core-3.23.1.jar
-rw-r--r--  1 rhusar  staff  1045444 Sep  5 12:24 aws-java-sdk-core-1.12.285.jar
-rw-r--r--  1 rhusar  staff   656333 Sep  5 12:24 aws-java-sdk-kms-1.12.285.jar
-rw-r--r--  1 rhusar  staff  1260486 Sep  5 12:24 aws-java-sdk-s3-1.12.285.jar
-rw-r--r--  1 rhusar  staff  3836661 Jun  4 17:26 byte-buddy-1.12.10.jar
-rw-r--r--  1 rhusar  staff   353793 Jan  2  2022 commons-codec-1.15.jar
-rw-r--r--  1 rhusar  staff    62050 Mar 19  2021 commons-logging-1.1.3.jar
-rw-r--r--  1 rhusar  staff    45024 Mar 15  2021 hamcrest-core-1.3.jar
-rw-r--r--  1 rhusar  staff   780321 Jan  2  2022 httpclient-4.5.13.jar
-rw-r--r--  1 rhusar  staff   328593 Mar 18  2021 httpcore-4.4.13.jar
-rw-r--r--  1 rhusar  staff   565410 Apr 20  2021 ion-java-1.0.2.jar
-rw-r--r--  1 rhusar  staff    75708 Mar  2  2022 jackson-annotations-2.12.6.jar
-rw-r--r--  1 rhusar  staff   365539 Mar  2  2022 jackson-core-2.12.6.jar
-rw-r--r--  1 rhusar  staff  1517298 Apr  1  2022 jackson-databind-2.12.6.1.jar
-rw-r--r--  1 rhusar  staff    61503 Mar 30  2022 jackson-dataformat-cbor-2.12.6.jar
-rw-r--r--  1 rhusar  staff  2238639 Oct 10 14:58 jgroups-5.2.7.Final.jar
-rw-r--r--  1 rhusar  staff    27591 Sep  5 12:24 jmespath-java-1.12.285.jar
-rw-r--r--  1 rhusar  staff   621931 Sep 26  2021 joda-time-2.8.1.jar
-rw-r--r--  1 rhusar  staff   384581 May 18  2021 junit-4.13.2.jar
```

you will notice that you also need to define a `scope` since e.g. junit is clearly not a *runtime* dependency :-)

Reply:
Well spotted on the missing scope, I can fix that.

I don't see any mention of copy-dependencies in the readme, which docs are you referring to

If I get a spare minute I want to submit a PR that has either a full module or at least bundle of cli to add the module

Reply:
> Well spotted on the missing scope, I can fix that.
> 
> I don't see any mention of copy-dependencies in the readme, which docs are you referring to

This was in reference to some infinispan server (red hat datagrid) documentation using the Maven copy-dependencies of a jgroups-extras project – see https://access.redhat.com/solutions/6048651

> If I get a spare minute I want to submit a PR that has either a full module or at least bundle of cli to add the module

A full module for what? WildFly?